### PR TITLE
Fix credential scan issues

### PR DESCRIPTION
**Pull request by TravisEz13**
Add suppression for lines with secrets.
Add suppressions for files which are secrets.
This will be consumed in the VSTS daily build.

Renamed tests with `ConvertTo-SecureString` to avoid false positives

### CODE REVIEW

**Review by SteveL-MSFT — APPROVED** [review]
LGTM

File: test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-SecureString.Tests.ps1 [hunk 1]
```diff
@@ -1,8 +1,9 @@
-Describe "ConvertTo-SecureString" -Tags "CI" {
+	Describe "ConvertTo--SecureString" -Tags "CI" {
```
Comment (adityapatwardhan) [inline test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-SecureString.Tests.ps1]: Seems to have an extra - between ConvertTo and SecureString

File: test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-SecureString.Tests.ps1 [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
 
-    Context "Checking return types of ConvertTo-SecureString" {
+    Context "Checking return types of ConvertTo--SecureString" {
```
Comment (adityapatwardhan) [inline test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-SecureString.Tests.ps1]: Seems to have an extra - between ConvertTo and SecureString

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

File: test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-SecureString.Tests.ps1 (hunk 2 above)
Comment (TravisEz13) [reply test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-SecureString.Tests.ps1]: See the previous comment

File: test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-SecureString.Tests.ps1 (hunk 1 above)
Comment (TravisEz13) [reply test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-SecureString.Tests.ps1]: I have to rename it to something other than convertto-securestring.

**Review by adityapatwardhan — APPROVED** [review]

**Comment by megamorf** [comment]
Just out of curiosity, is the `CS002:SecretInNextLine` suppression VSTS specific? Haven't seen that before.

**Comment by TravisEz13** [comment]
@megamorf It's specific to the tool we use.  Unfortunately, the tool itself is internal to Microsoft only.  That is why we run it in VSTS and not are normal CI systems.

**Comment by megamorf** [comment]
@TravisEz13 Ah, that makes sense. Thanks for the answer :-)

**Pull request merged**